### PR TITLE
Continue replacing duplicate ecma definitions

### PR DESCRIPTION
**Pull request by Achie72**
JerryScript-DCO-1.0-Signed-off-by: Bela Toth tbela@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/ecma/base/ecma-gc.c [hunk 1]
```diff
@@ -2030,9 +2030,9 @@ ecma_free_unused_memory (jmem_pressure_t pressure) /**< current pressure */
   if (JERRY_LIKELY (pressure == JMEM_PRESSURE_LOW))
   {
 #if JERRY_PROPRETY_HASHMAP
-    if (JERRY_CONTEXT (ecma_prop_hashmap_alloc_state) > ECMA_PROP_HASHMAP_ALLOC_ON)
+    if (JERRY_CONTEXT (JERRY_PROP_hashmap_alloc_state) > JERRY_PROP_HASHMAP_ALLOC_ON)
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-gc.c:2033]: The name of a variable should not be changed. Similar cases below.

File: jerry-core/ecma/base/ecma-helpers-string.c [hunk 2]
```diff
@@ -236,13 +236,13 @@ ecma_new_symbol_from_descriptor_string (ecma_value_t string_desc) /**< ecma-stri
  *         false - otherwise
  */
 bool
-ecma_prop_name_is_symbol (ecma_string_t *string_p) /**< ecma-string */
+JERRY_PROP_name_is_symbol (ecma_string_t *string_p) /**< ecma-string */
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers-string.c:239]: Also wrong.

**Review by zherczeg — CHANGES_REQUESTED** [review]

File: jerry-core/ecma/base/ecma-globals.h [hunk 3]
```diff
@@ -1942,12 +1924,12 @@ typedef struct
  * If ecma_prop_hashmap_alloc_state other other than this value, it is
  * disabled.
  */
-#define ECMA_PROP_HASHMAP_ALLOC_ON 0
+#define JERRY_PROP_HASHMAP_ALLOC_ON 0
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-globals.h:1927]: These should not be changed, only those which are defined in jerryscript-types.h

**Review by zherczeg — CHANGES_REQUESTED** [review]
Better

**Review by zherczeg — APPROVED** [review]
LGTM

**Review by rerobika — APPROVED** [review]

**Pull request merged**